make/photon/prepare/templates/trivy-adapter/env.jinja

```diff
@@ -0,0 +1,7 @@
+SCANNER_LOG_LEVEL={{log_level}}
+SCANNER_STORE_REDIS_URL=redis://redis:6379
```

Comment:
@heww I need your help to use variables for redis URLs instead of hardcoding them

Reply:
We can add `redis_url_trivy` in the result of `get_redis_configs` function in `make/photon/prepare/utils/configs.py` so that here can render SCANNER_STORE_REDIS_URL to `{{ redis_url_trivy }}`.

And in `harbor.yml.tmpl` add a default redis db index for trivy in `external_redis`.

```
# external_redis:
#   host: redis
#   port: 6379
#   password:
#   # db_index 0 is for core, it's unchangeable
#   registry_db_index: 1
#   jobservice_db_index: 2
#   chartmuseum_db_index: 3
#   clair_db_index: 4
```

Reply:
> We can add `redis_url_trivy` in the result of `get_redis_configs` function in `make/photon/prepare/utils/configs.py` so that here can render SCANNER_STORE_REDIS_URL to `{{ redis_url_trivy }}`.
> 
> And in `harbor.yml.tmpl` add a default redis db index for trivy in `external_redis`.
> 
> ```
> # external_redis:
> #   host: redis
> #   port: 6379
> #   password:
> #   # db_index 0 is for core, it's unchangeable
> #   registry_db_index: 1
> #   jobservice_db_index: 2
> #   chartmuseum_db_index: 3
> #   clair_db_index: 4
> ```

Done!